.gitignore

```diff
@@ -1,5 +1,9 @@
 # ROOT
 /node_modules
+cypress.config.ts
+
+# Cypress
+/cypress/support/commands.ts
```

Comment:
this file must not exist, there is no reason to have cypress in the root

Reply:
Removed this root/cypress folder